### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/bubbletea/commit/c5946907ce919dbab5829b924596867e4c6912b7#commitcomment-86423265

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/bubbletea/commit/84401d75be7b0844fac522b05d69796ed45b4464
 
Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: screen.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func DisableMouse() Msg {
 	return disableMouseMsg{}
 }
 
-// disableMouseMsg is an internal message that that signals to stop listening
+// disableMouseMsg is an internal message that signals to stop listening
```
Comment (jsoref) [inline screen.go:93]: Sometimes doubled `that` is intentional/necessary. This didn't feel like one of those cases.

**Review by jsoref — COMMENTED** [review]
Spelling corrections were suggested by Google Sheets. Duplicate corrections are by me. All fault mine.

File: examples/pager/artichoke.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ A casual introduction. 你好世界!
 
 The _artichoke_ is mentioned as a garden plant in the 8th century BC by Homer
 **and** Hesiod. The naturally occurring variant of the artichoke, the cardoon,
-which is native the the Mediterranean area, also has records of use as a food
+which is native the Mediterranean area, also has records of use as a food
```
Comment (muesli) [inline examples/pager/artichoke.md:10]: Should probably read "which is native to the Mediterranean area".

**Comment by muesli** [comment]
Thank you @jsoref! Looking at some of them, I can't believe I've never noticed any of these dupes :smile:

**Comment by muesli** [comment]
Thank you, @jsoref! I'll check out what the spell-checker looks like in action. Sounds intriguing.

**Pull request merged**